src/mqt/predictor/rl/helper.py

```diff
@@ -571,3 +583,22 @@ def get_device_index_of_device(device_name: str) -> int:
 
     msg = "No suitable device found."
     raise RuntimeError(msg)
+
+
+def final_layout_pytket_to_qiskit(pytket_circuit: Circuit) -> Layout:
+    pytket_layout = pytket_circuit.qubit_readout
+    size_circuit = pytket_circuit.n_qubits
+    qiskit_layout = Layout()
+    qiskit_qreg = QuantumRegister(size_circuit, "q")
+
+    pytket_layout = dict(sorted(pytket_layout.items(), key=lambda item: item[1]))
+
+    for node, qubit_index in pytket_layout.items():
+        new_index = node.index[0]
+        qiskit_layout[new_index] = qiskit_qreg[qubit_index]
+
+    for i in range(size_circuit):
+        if i not in set(pytket_layout.values()):
+            qiskit_layout[i] = qiskit_qreg[i]
```

Comment:
Another (probably stupid) question: these are qubits that are not measured at all. Does Qiskit still include information about these in its final layout?

Reply:
Yes, it does.